recipes/install_linux.rb

```diff
@@ -1,8 +1,24 @@
-yum_repository 'gauge-stable' do
-  description 'Gauge stable repository'
-  baseurl "http://dl.bintray.com/gauge/gauge-rpm/gauge-stable"
-  gpgcheck false
-  action :create
+case node['platform_family']
+when 'rhel', 'centos'
```

Comment:
See https://github.com/chef/ohai/blob/cb90ed45d4e8afa4e5899222a78b201e1d967a53/lib/ohai/plugins/linux/platform.rb#L119 for possible `platform_family` values.

You'd use `when 'rhel', 'fedora', 'amazon'` and `when 'debian', 'ubuntu'`